types/styled-components/test/index.tsx

```diff
@@ -19,6 +19,7 @@ import styled, {
     FlattenInterpolation,
 } from 'styled-components';
 import {} from 'styled-components/cssprop';
+import {} from 'styled-components/test-utils';
```

Comment:
What's this for? Did you forget to include actual tests?

Reply:
Added tests in next commit (with some modifications on the files)